cime_config/testdefs/testmods_dirs/clm/collapse_pfts_78_to_16/user_nl_clm

```diff
@@ -0,0 +1,3 @@
+for_testing_allow_non_annual_changes = .true.
+fsurdat = '/glade/p/cesmdata/cseg/inputdata/lnd/clm2/surfdata_map/surfdata_10x15_78pfts_CMIP6_simyr1850_c170824.nc'
```

Comment:
Please add a comment about the origin of these files. e.g., are these the files that are used out-of-the-box in some other configuration? If so, please state something like, "These are the files that are used out-of-the-box for compset X, resolution Y". This is important to keep this test maintainable as we introduce new surface datasets: we need to know how to update this testmod.

It would also be helpful if the README file for this test pointed out that it only works at f10 resolution.

Reply:
You should also change the name of the test to have "f10" in the name, so it's obvious that it can only work at f10 resolution. That's something we've done in the past when we have a resolution dependent test.

Reply:
Bill: I added comments in the README following these suggestions, though I'm not sure whether they are you had in mind. Pls confirm after next commit.

Erik: This is the test name that I added in testlist_clm.xml:
<test name="ERS_D_Ld10" grid="f10_f10_musgs" compset="IHistClm50SpG" testmods="clm/collapse_pfts_78_to_16">
so the f10 shows up in the grid entry. Are you recommending that it should appear elsewhere, as well?

Reply:
What Erik meant, I think, is that the testmod should have f10 in the name.

Reply:
Done in this commit:
0ed02f6